### PR TITLE
File Added: Boostrap assignment 3, 4

### PR DESCRIPTION
**Pull request by vijaya1811**

### CODE REVIEW

File: vijaya/Boostrap/assignment3/assignment3.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+                        <div class="col-md-3"><img src="anita.png" class="rounded-circle circle2" alt="marry">
+                        </div>
+                    </div>
+                    <div class="col-md-3 text-justify">
+                        <h4 class="heading_color">ANITA SMITH</h4>
+                        Downloading this awesome diagram.Bring your presentation to life.Capture your audience attention.
+                    </div>
+                </div>
+            </div>
+        </body>
```
Comment (Pawan1005) [inline vijaya/Boostrap/assignment3/assignment3.html]: html tag closing missing

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

File: vijaya/Boostrap/assignment3/assignment3.html (hunk 1 above)
Comment (shubhamdalal49) [inline vijaya/Boostrap/assignment3/assignment3.html]: where is html tag

**Review by shubhamdalal49 — CHANGES_REQUESTED** [review]

File: vijaya/Boostrap/assignment3/assignment3.html (hunk 1 above)
Comment (vijaya1811) [reply vijaya/Boostrap/assignment3/assignment3.html]: done changes sir.

Comment (vijaya1811) [reply vijaya/Boostrap/assignment3/assignment3.html]: done.

File: vijaya/Boostrap/assignment3/assignment3.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            }
+        </style>
+    </head>
+    <body>
+        <h1 class="text-center"><span class="heading">About</span> Us</h1>
+        <hr>
+        <hr>
+        <p style="text-align: center;">Downloading this awesome diagram.Bring your presentation to life.Capture your audience attention.</p><br>
+        <div class="container">
+             <div class="row">
```
Comment (nileshp1122) [inline vijaya/Boostrap/assignment3/assignment3.html]: check indentation extra space in this line

File: vijaya/Boostrap/assignment3/assignment3.html [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+                <div class="col-md-3 text-justify">
+                    <h4 class="heading_color">JOHN SMITH</h4>
+                    Downloading this awesome diagram.Bring your presentation to life.Capture your audience attention..</div>
+                    <div class="green">
+                    <div class="col-md-3"><img src="john1.jpeg" class="rounded-circle float-left circle" alt="marry"></div></div>
+                    <div class="green2">
+                    <div class="col-md-3"><img src="marry.jpeg" class="rounded-circle float-right circle2" alt="marry"></div>
+                </div>
+                 <div class="col-md-3 text-justify">
```
Comment (nileshp1122) [inline vijaya/Boostrap/assignment3/assignment3.html:92]: check indentation

File: vijaya/Boostrap/assignment3/assignment3.html [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+                    <h4 class="heading_color1">MARRY SMITH</h4>
+                    Downloading this awesome diagram.Bring your presentation to life.Capture your audience attention.</div>
+                   </div><br><br>
```
Comment (nileshp1122) [inline vijaya/Boostrap/assignment3/assignment3.html:95]: check indentation

File: vijaya/Boostrap/assignment3/assignment3.html [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+                   <div class="row">
```
Comment (nileshp1122) [inline vijaya/Boostrap/assignment3/assignment3.html:96]: check indentation on space

**Review by nileshp1122 — CHANGES_REQUESTED** [review]

File: vijaya/Boostrap/assignment3/assignment3.html (hunk 2 above)
Comment (vijaya1811) [reply vijaya/Boostrap/assignment3/assignment3.html]: ok done

**Review by Pawan1005 — APPROVED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

**Review by urmilagagare — APPROVED** [review]

**Pull request merged**